unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/Tests/WearableGridControllerShould.cs

```diff
@@ -444,7 +444,7 @@ public void ChangePageWhenViewRequestsIt()
                                         Arg.Any<string>(),
                                         Arg.Any<(NftOrderByOperation type, bool directionAscendent)?>());
 
-            view.Received(1).SetWearablePages(3, 4);
+            view.Received(1).SetWearablePages(4);
```

Comment:
we are not validating in the tests that the `GoToPage` method is being called

Reply:
The `GoToPage` function was removed in the first feedback 👍